tests/verror.unit.js

```diff
@@ -1,6 +1,7 @@
+/* eslint no-unused-expressions: 0 */
```

Comment:
Why don't we just remove all the unused expressions/dead code?

Reply:
done!